### PR TITLE
Docker compose code executor

### PR DESCRIPTION
**Pull request by demitrin**

### CODE REVIEW

File: docker-compose-workflows.yml [hunk 1]
```diff
@@ -97,15 +100,31 @@ services:
...
       - backend-network
       - db-connector-network
       - workflows-network
+      - code-executor-network
+    restart: on-failure
+
+  code-executor:
+    build:
+      context: ./
+      dockerfile: DockerfileCodeExecutor
```
Comment (roberto-retool) [inline docker-compose-workflows.yml:118]: ultra nit: I believe naming convention would have this be `CodeExecutor.Dockerfile`

File: docker-compose-workflows.yml [hunk 2]
```diff
@@ -14,19 +14,22 @@ services:
...
       - WORKFLOW_TEMPORAL_CLUSTER_FRONTEND_PORT=7233
       - WORKFLOW_BACKEND_HOST=http://api:3000
+      - CODE_EXECUTOR_INGRESS_DOMAIN=http://code-executor:3004
     networks:
       - frontend-network
       - backend-network
       - db-connector-network
       - db-ssh-connector-network
       - workflows-network
+      - code-executor-network
```
Comment (roberto-retool) [inline docker-compose-workflows.yml:24]: do we need to also add this to `workflow-worker` container? or do we never call code-executor directly from the worker

**Review by roberto-retool — APPROVED** [review]
🔥 
couple comments. lgtm after

File: docker-compose-workflows.yml (hunk 2 above)
Comment (demitrin) [reply docker-compose-workflows.yml:24]: wait i think this is done already

**Pull request merged**